libs/components/ag-grid/src/lib/modules/ag-grid/types/cell-type.ts

```diff
@@ -69,23 +69,23 @@ export enum SkyCellType {
   /**
    * **Edit and read-only modes**
    * <br/>
-   * Specifies that cells in the column will render as [SKY UX checkbox components](https://developer.blackbaud.com/skyux/components/checkbox). It allows the user to select multiple rows, and adds a highlight to selected rows. The [Ag Grid `rowNode`](https://www.ag-grid.com/javascript-grid-row-node/) will be updated to reflect the selected state.
+   * That cells in the column will render as [SKY UX checkbox components](https://developer.blackbaud.com/skyux/components/checkbox). It allows the user to select multiple rows, and adds a highlight to selected rows. The [Ag Grid `rowNode`](https://www.ag-grid.com/javascript-grid-row-node/) will be updated to reflect the selected state.
    */
   RowSelector = 'skyCellRowSelector',
   /**
    * **Edit mode**
    * <br/>
-   * Specifies that cells in the column will be edited as HTML text `inputs`. Any characters can be entered.
+   * That cells in the column will be edited as HTML text `inputs`. Any characters can be entered.
    * <br/><br/>
    * **Read-only mode**
    * <br/>
-   * Specifies cells in the column will render as their string value.
+   * Cells in the column will render as their string value.
    */
   Text = 'skyCellText',
   /**
    * **Edit and read-only modes**
    * <br/>
-   * Specifies cell in the column will be passed to a validator function which flags erroneous entries. You can set the validator function and message by passing them to [column definition's `cellRendererParams` property](https://www.ag-grid.com/angular-data-grid/column-properties/#reference-editing) as `skyComponentProperties.validator` and `skyComponentProperties.validatorMessage`. SkyCellType.Validator can be combined with other cell types, such as SkyCellType.Autocomplete or SkyCellType.Date, by using the array syntax for the [column definition's `type` property](https://www.ag-grid.com/angular-data-grid/column-properties/#reference-editing).
+   * Cell in the column will be passed to a validator function which flags erroneous entries. You can set the validator function and message by passing them to [column definition's `cellRendererParams` property](https://www.ag-grid.com/angular-data-grid/column-properties/#reference-editing) as `skyComponentProperties.validator` and `skyComponentProperties.validatorMessage`. SkyCellType.Validator can be combined with other cell types, such as SkyCellType.Autocomplete or SkyCellType.Date, by using the array syntax for the [column definition's `type` property](https://www.ag-grid.com/angular-data-grid/column-properties/#reference-editing).
```

Comment:
```suggestion
   * Cells in the column will be passed to a validator function that flags erroneous entries. You can set the validator function and message by passing them to [column definition's `cellRendererParams` property](https://www.ag-grid.com/angular-data-grid/column-properties/#reference-editing) as `skyComponentProperties.validator` and `skyComponentProperties.validatorMessage`. SkyCellType.Validator can be combined with other cell types, such as SkyCellType.Autocomplete or SkyCellType.Date, by using the array syntax for the [column definition's `type` property](https://www.ag-grid.com/angular-data-grid/column-properties/#reference-editing).
```
I assume this should be plural like all the others. (Also, `which` should be `that` because it's a dependent clause.)

Reply:
Can we make these tweaks here? Or should I to file a bug to clean this up separately?